tempto-core/src/main/java/io/prestosql/tempto/internal/configuration/YamlConfiguration.java

```diff
@@ -43,8 +45,13 @@ public YamlConfiguration(InputStream yamlInputStream)
 
     private MapConfiguration loadConfiguration(String yamlString)
     {
+        requireNonNull(yamlString, "yamlString is null");
         Yaml yaml = new Yaml();
         Object loadedYaml = yaml.load(yamlString);
+        if (loadedYaml == null) {
```

Comment:
wow... I would never guess that it might return `null` in such case.

Reply:
@lukasz-walkiewicz told me :)